appinfo/routes/routesBanController.php

```diff
@@ -17,6 +17,6 @@
 		/** @see \OCA\Talk\Controller\BanController::listBans() */
 		['name' => 'Ban#listBans', 'url' => '/api/{apiVersion}/ban/{token}', 'verb' => 'GET', 'requirements' => $requirements],
 		/** @see \OCA\Talk\Controller\BanController::unbanActor() */
-		['name' => 'Ban#unbanActor', 'url' => '/api/{apiVersion}/ban/{token}', 'verb' => 'DELETE', 'requirements' => $requirements],
+		['name' => 'Ban#unbanActor', 'url' => '/api/{apiVersion}/ban/{token}/{banId}', 'verb' => 'DELETE', 'requirements' => $requirements],
```

Comment:
Don't forget about endpoint here:
https://github.com/nextcloud/spreed/blob/04b1d46a41affedd2065d98674beface67e437a3/src/services/banService.ts#L45-L50

```js
const unbanActor = async function(token: string, banId: unbanActorParams['banId'], options?: object): unbanActorResponse {
	return axios.delete(generateOcsUrl('/apps/spreed/api/v1/ban/{token}/{banId}', { token, banId }, options), options)
}
```

And pay attention, if there would be openapi changes because of that

Reply:
> if there would be openapi changes because of that

There will be and have to be :( I'm sorry about that.